modules/statics/src/networks.ts

```diff
@@ -834,6 +834,18 @@ class RuneTestNet extends Testnet implements AccountNetwork {
   explorerUrl = 'https://runescan.io/tx/?network=stagenet';
 }
 
+class Baby extends Mainnet implements AccountNetwork {
+  name = 'Babylon';
+  family = CoinFamily.BABY;
+  explorerUrl = '';
+}
+
+class BabyTestnet extends Testnet implements AccountNetwork {
+  name = 'Testnet Babylon';
+  family = CoinFamily.BABY;
+  explorerUrl = 'https://testnet.babylon.explorers.guru/';
```

Comment:
```suggestion
  explorerUrl = 'https://testnet.babylon.explorers.guru/transaction/';
```
It should be transactions URL. Applies to mainnet class as well